### PR TITLE
Remove 2 vote difference requirement

### PR DESCRIPTION
**Pull request by skalnik**
> "Ask any racer. It doesn't matter if you win by an inch or a mile, winning is winning."
> —Vin Diesel

— Mike ":car: :checkered_flag: :blue_car: " Skalnik

### CODE REVIEW

**Comment by sshirokov** [comment]
The check needs to stay, if you want to fuck with it change the constant

**Comment by skalnik** [comment]
@sshirokov Updated :white_check_mark:

File: scripts/vote.coffee [hunk 1]
```diff
@@ -49,8 +49,8 @@ class Vote
     return if @finished or @expired
     # We're not done unless we get three votes
     return if (@votes.yes.length + @votes.no.length) < 3
-    # We're not done if it's only a two vote difference
-    return if (Math.abs(@votes.yes.length - @votes.no.length) < 2)
+    # We're not done if there's no difference between the votes
+    return if (Math.abs(@votes.yes.length - @votes.no.length) < 0)
```
Comment (sshirokov) [inline scripts/vote.coffee]: You mean `< 1` buddy.

File: scripts/vote.coffee [hunk 2]
```diff
@@ -49,8 +49,8 @@ class Vote
     return if @finished or @expired
     # We're not done unless we get three votes
     return if (@votes.yes.length + @votes.no.length) < 3
-    # We're not done if it's only a two vote difference
-    return if (Math.abs(@votes.yes.length - @votes.no.length) < 2)
+    # We're not done if there's no difference between the votes
+    return if (Math.abs(@votes.yes.length - @votes.no.length) <= 0)
```
Comment (sshirokov) [inline scripts/vote.coffee]: I mean, that's the least readable way to express that :crying_cat_face:

**Comment by skalnik** [comment]
@sshirokov Any other thoughts here?

**Pull request merged**